flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/functions/casting/CastRuleUtils.java

```diff
@@ -43,6 +43,10 @@ static String nullLiteral(boolean legacyBehaviour) {
         return legacyBehaviour ? strLiteral("null") : strLiteral("NULL");
     }
 
+    static String operator(String operator, Object argLeft, Object argRight) {
```

Comment:
Can we use "left", "operator", "right" rather than the infix?

Reply:
This is still not addressed

Reply:
Sorry, I messed up something when I removed that commit, I will fix and ping when ready.